tests/storage/test_mount_sanity.py

```diff
@@ -0,0 +1,74 @@
+import threading
+import subprocess
+import time
+import sys
+import _thread as thread
+import os
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+import argparse
+
+def get_args():
```

Comment:
should we allow a specific mountpoint to be tested?

Reply:
I added mountpoint as a parameter